exporters/otlp/src/otlp_grpc_exporter.cc

```diff
@@ -68,15 +67,25 @@ std::unique_ptr<proto::collector::trace::v1::TraceService::Stub> MakeServiceStub
   {
     channel = grpc::CreateChannel(grpc_target, grpc::InsecureChannelCredentials());
   }
-  return proto::collector::trace::v1::TraceService::NewStub(channel);
+
+  return channel;
+}
+
+/**
+ * Create service stub to communicate with the OpenTelemetry Collector.
+ */
+std::unique_ptr<proto::collector::trace::v1::TraceService::Stub> MakeTraceServiceStub(
```

Comment:
Why to split this function ?

Reply:
Split of `MakeTraceServiceStub` was meant to move `MakeGrpcChannel` to common lib, but then it also adds gRPC dependence to the common lib. I suggest we improve this in a separate PR.